### PR TITLE
Added log gathering to the list of availble operations

### PR DESCRIPTION
**Pull request by themurph**
/cc @jupierce

### CODE REVIEW

File: tower-scripts/bin/cicd-control.sh [hunk 1]
```diff
@@ -212,6 +212,14 @@ elif [ "${OPERATION}" == "status" ]; then
     /usr/local/bin/autokeys_loader ./aos-cd-cluster-status.sh ${CLUSTERNAME}
   popd
 
+################################################
+# CLUSTER LOG GATHERING
+################################################
+elif [ "${OPERATION}" == "logs" ]; then
```
Comment (jupierce) [inline tower-scripts/bin/cicd-control.sh]: pushd/popd and potentially others output to stdout which could corrupt the tarball generated by gather-logs.sh . Need to move this up the script and exit 0 before more content it output.

File: tower-scripts/bin/cicd-control.sh [hunk 2]
```diff
@@ -212,6 +212,14 @@ elif [ "${OPERATION}" == "status" ]; then
...
+
+  # Gather the logs for the specified cluster
+  ./gather_logs.sh ${CLUSTERNAME}
```
Comment (jupierce) [inline tower-scripts/bin/cicd-control.sh]: gather-logs.sh , not gather_logs.sh

**Comment by themurph** [comment]
Updated title to reflect the correct commit

File: tower-scripts/bin/cicd-control.sh (hunk 1 above)
Comment (themurph) [inline tower-scripts/bin/cicd-control.sh]: Done

File: tower-scripts/bin/cicd-control.sh (hunk 2 above)
Comment (themurph) [inline tower-scripts/bin/cicd-control.sh]: That's embarrassing

**Comment by themurph** [comment]
@kargakis Added usage for the logging portion of the script

File: tower-scripts/bin/cicd-control.sh [hunk 3]
```diff
@@ -55,6 +60,16 @@ export OPERATION=$2
...
 
+################################################
+# CLUSTER LOG GATHERING
+# PLEASE DO NOT ADD ANY NEW OPERATIONS BEFORE HERE
+################################################
+if [ "${OPERATION}" == "logs" ]; then
+
+  # Gather the logs for the specified cluster
+  ./gather-logs.sh ${CLUSTERNAME}
+fi
```
Comment (jupierce) [inline tower-scripts/bin/cicd-control.sh]: Looks like this will fall through and corrupt the tarball with extraneous stdout.

**Pull request merged**